### DISCUSSION TITLE
SaintCoinach解包出错

### DISCUSSION THREAD

**Discussion by lincTHU** [GpointChen/FFXIVChnTextPatch-GP#292] [General]
![image](https://user-images.githubusercontent.com/8018928/213870820-4362fc4c-b620-42b2-9e6f-344235e51422.png)
使用大佬修改的hexcode版SaintCoinach解包最新6.3包体的时候出错
Definition已经更新到XIVAPI最新发布的版本了
请问有解决方法吗（）

**Comment by GpointChen** [comment]
這個我用原版的SC也會跳錯

我目前也還沒時間找問題在哪，可能等之後看xivapi會不會修正吧
